ihatemoney/migrations/versions/b9a10d5d63ce_.py

```diff
@@ -49,6 +49,7 @@ def upgrade():
         sa.Column("date", sa.Date(), nullable=True),
         sa.Column("what", sa.UnicodeText(), nullable=True),
         sa.Column("archive", sa.Integer(), nullable=True),
+        sa.Column("is_reimbursement", sa.Boolean(), nullable=True),
```

Comment:
This file is auto-generated, please read https://ihatemoney.readthedocs.io/en/latest/contributing.html#creating-database-migrations